src/modules/site-v2/templates/releases/download_tab_strain_v2_issues.html

```diff
@@ -1,4 +1,12 @@
 
+{% set ns = namespace(hasIssue=false) %}
+{% for strain in strain_listing_issues %}
+    {% if strain.issues == True %}
+        {% set ns.hasIssue=true %}
+    {% endif %}
+{% endfor %}
+
+{% if ns.hasIssue == true %}

```

Comment:
prefer the capitalization for boolean types (`True` and `False`). In this case, we can short-circuit as well, by only using:
```
{% if ns.hasIssue %}
```